configuration/connect-to-ms-exchange.md

```diff
@@ -0,0 +1,233 @@
+# How to connect to Microsoft Exchange with Qubes OS
+
+## Introduction
+This Howto describes how you can connect to your Exchange Server with Qubes OS.
+It is based on using a Exchange-to-IMAP-Gateway which is named Davmail.
+
+The howto covers:
+
+- Installation/Configuration of Davmail
+- Configuration to use Thunderbird
+- Configuration to use neomutt
+
+the following VMs will be mentioned in this howto:
+
+- t-mail = TemplateVM
+- my-vault = ApppVM which is used for SplitGPG
+- my-workmail = AppVM
+
+TODO:
+
+- Howto use SplitGPG with neomutt
+- Howto work with attachments (PDF/Pictures) and HTML mails in neomutt
+
+## Davmail
+
+install required packages for Davmail
+
+`sudo dnf install java-9-openjdk`
+
+Download Davmail in an AppVM and qvm-copy-to-vm it over to the mail template vm
+in the AppVM which has internet access:
+
+`wget https://kent.dl.sourceforge.net/project/davmail/davmail/4.8.3/davmail-linux-x86_64-4.8.3-2554.tgz`
+`qvm-copy-to-vm t-mail davmail-linux-x86_64-4.8.3-2554.tgz`  
+
+In the TemplateVM:
+
+`sudo tar -xvzf ~/QubesIncoming/<AppVM>/davmail.*.tgz -C /opt`
+`sudo mv /opt/davmail* /opt/davmail`
+
+tell your template VM where to find the GPG Vault VM 
+
+`export QUBES_GPG_DOMAIN=my-vault`
+
+test access via SplitGPG (this command should list your secret-keys in the Vault-AppVM)
+
+`qubes-gpg-client -K` 
+
+
+Shutdown TemplateVM and create a new AppVM based on the TemplateVM
+in dom0
+
+`qvm-shutdown --wait t-mail`
+`qvm-create --template t-mail --label blue my-workmail`
+
+Open Qubes Setting for the new AppVM and increase storage capacity
+Private storage max. size: 20480 MiB
+
+Start AppVM and continue configuration there
+
+`qvm-run --auto my-workmail gnome-terminal`
+
+Start davmail
+
+`/opt/davmail/davmail.sh &`
+
+- Main Tab: Echange Protocol: Auto
+- Main Tab: OWA (Exchange) URL: https://owa.domain.com/owa/
+- Main Tab: [ ] Local POP port
+- Advanced Tab: Default windows domain: <YOUR-DOMAIN>
+- Advanced Tab: [x] Disable Update Check
+
+tell your AppVM VM where to find the GPG Vault VM 
+
+`echo "export QUBES_GPG_DOMAIN=my-vault" >> /home/user/.bashrc`
```

Comment:
Better write `my-vault` to `/rw/config/gpg-split-domain` (which is sourced by `/etc/profile.d/qubes-gpg.sh`). This way it will work seamlessly with `qubes-gpg-client-wrapper`.